drift/views/v1.py

```diff
@@ -57,6 +67,30 @@ def ensure_account_number():
                             message="account number not found on identity token")
 
 
+@section.before_app_request
+def ensure_entitled():
+    """
+    check if the request is entitled. We run this on all requests and bail out
+    if the URL is whitelisted.
+    """
+    # TODO: Blueprint.before_request was not working as expected, using
+    # before_app_request and checking URL here instead.
+    if _is_mgmt_url(request.path):
+        return  # allow request
```

Comment:
Does python's `return` return `true` by default?

Reply:
it returns `None`. If you return a value, Flask will return that to the user instead of doing further processing.

http://flask.pocoo.org/docs/1.0/api/#flask.Flask.before_request has more info